.eslintrc.json

```
@@ -16,6 +16,6 @@
         "@typescript-eslint"
     ],
     "rules": {
-        "max-len": ["error", { "code": 100 }]
+        "max-len": ["error", { "code": 200 }]
```

i think 100 chars is enough
```suggestion
        "max-len": ["error", { "code": 100 }]
```